tests/testthat/test_compactness.R

```diff
@@ -5,52 +5,56 @@ test_that('compactness works', {
-  ), ConvexHull = c(
-    0.68886719971328,
-    0.657283200146699, 0.696409105212057, 0.724494964993755, 0.657283200146699,
-    0.598769626173265
-  ), Reock = c(
-    0.43437300523785, 0.306645852527558,
-    0.224742497097297, 0.424640741237631, 0.306645852527558, 0.210845488809258
-  ), BoyceClark = c(
-    0.855392779807052, 0.750250900412383, 0.735085159873138,
-    0.832201015045522, 0.750250900412383, 0.804636547547239
-  ), FryerHolden = c(
-      1469127189558785024, 1469127189558785024, 1469127189558785024,
-      1192801089849208320, 1192801089849208320, 1192801089849208320
-  ), EdgesRemoved = c(
-    16, 16, 16, 16, 16, 16
-  ), FracKept = c(
-    0.6862745, 0.6862745,
-    0.6862745, 0.6862745, 0.6862745, 0.6862745
-  ), logSpanningTree = c(
-    12.6907185515128, 12.6907185515128, 12.6907185515128,
-    12.6907185515128, 12.6907185515128, 12.6907185515128
-  ), draw = c(
-    1,
-    1, 1, 2, 2, 2
-  )), row.names = c(NA, -6L), class = c(
-    'tbl_df',
-    'tbl', 'data.frame'
-  ))
+    expected <- structure(list(
+      district = c(1L, 2L, 3L, 1L, 2L, 3L), PolsbyPopper = c(
+        0.28097417163596,
+        0.25130532031779, 0.32157642432607, 0.394022094408435, 0.25130532031779,
+        0.223058240647906
+      ), Schwartzberg = c(
+        0.530069968622974, 0.501303620890364,
+        0.567077088521543, 0.627711792472019, 0.501303620890364, 0.472290419813811
+      ), LengthWidth = c(
+        0.926285160038819, 0.653935724266415, 0.404149212048674,
+        0.874431619447365, 0.653935724266415, 0.417276720351394
+      ), ConvexHull = c(
+        0.688881302064597,
+        0.657235285584608, 0.696584187586064, 0.724528346382683, 0.657235285584608,
+        0.598835368827732
+      ), Reock = c(
+        0.436913044036918, 0.308475061161299,
+        0.226034757924429, 0.427167165858294, 0.308475061161299, 0.21214911798771
+      ), BoyceClark = c(
+        0.820211159002851, 0.761493024898368, 0.729357474768565,
+        0.834576569326994, 0.761493024898368, 0.836077179813682
+      ), FryerHolden = c(
+        131124756.627412,
+        131124756.627412, 131124756.627412, 105282505.012801, 105282505.012801,
+        105282505.012801
+      ), EdgesRemoved = c(16, 16, 16, 16, 16, 16),
+      FracKept = c(
+        0.843137254901961, 0.843137254901961, 0.843137254901961,
+        0.843137254901961, 0.843137254901961, 0.843137254901961
+      ),
+      logSpanningTree = c(
+        12.6907185515128, 12.6907185515128, 12.6907185515128,
+        12.6907185515128, 12.6907185515128, 12.6907185515128
+      ), draw = c(
+        1,
+        1, 1, 2, 2, 2
+      )
+    ), row.names = c(NA, -6L), class = c(
+      'tbl_df',
+      'tbl', 'data.frame'
+    ))
 
   expect_equal(comp, expected, tolerance=1e-4)
 })
 
 
 test_that('test prep polsbypopper works', {
-  test <- redist.prep.polsbypopper(shp = fl25)
+  test <- redist.prep.polsbypopper(shp = fl25) %>% suppressWarnings()
```

Comment:
what's the warning?

Reply:
Was the missing epsg warning because the correct way `epsg = FALSE` didn't work until our recent update. Fixed now.